cbmc/proofs/IotMqtt_Cleanup/Makefile

```diff
@@ -0,0 +1,10 @@
+ENTRY=IotMqtt_Cleanup
+
+ABSTRACTIONS += --remove-function-body IotLog_Generic
```

Comment:
You had a nice comment you added to the _init makefile in response to DSN, maybe we should use that everywhere.

Reply:
Done!